Assets/Scripts/Minigames/CowardMiniGame/LightManager.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LightManager : MonoBehaviour
+{
+    [SerializeField] private float _blinkingSpeed;
+    [SerializeField] GameObject[] _lights;
+    private bool _isFlashing;
+    private int _currentLight = 0;
+
+    private void OnEnable()
+    {
+        RipcordBehavior.OnRipcordScore += OnScore;
+        RipcordBehavior.OnPlayerDetectionChange += SetIsFlashing;
+    }
+
+    private void OnDisable()
+    {
+        RipcordBehavior.OnRipcordScore -= OnScore;
+        RipcordBehavior.OnPlayerDetectionChange -= SetIsFlashing;
+    }
+
+    public void StartBlinkingLightCoroutine()
+    {
+        StartCoroutine(BlinkingLights());
```

Comment:
this is a small thing but I think it would be better to have the light blinking or what visually shows the light intensity changing to follow the ripcord rather than the player since sometime you can be right infront of the ripcord a bit and it intensifies and blinks early.

Reply:
It is based off the ripcord! This coroutine is attached to the OnPlayerDetectionChange which is called when the Ripcord collides with the Release trigger. I will rename that event